Automattic/woocommerce-payments
# Add compatibility between Multi-Currency and WooCommerce Pre-Orders.

Fixes #3275

#### Changes proposed in this Pull Request

* Add new `WooCommercePreOrders` class.
* Add filter `wc_pre_orders_fee` to convert the pre-order fee for a product.

#### Testing instructions

* Install WooCommerce Pre-Orders, which is available in the all-plugins repo.
* Create a new product, making sure to set it as a pre-order under the _Pre-Orders_ tab in the product.
* Also add a _Pre-Order Fee_ to the product.
* View product on front end and make sure cost is converted.
* Add to cart, and then view the cart.
* Make sure the Pre-Order Fee is converted in the cart.
* Check out and make sure costs in the order are converted.

-------------------


**Post merge**
